driver/config.yaml

```diff
@@ -0,0 +1,9 @@
+experiment_dir: '/scratch1/NCEPDEV/da/Kevin.Dougherty/test/'
```

Comment:
not needed now, but if we ever 'operationalize' this, we will need to use the standard naming conventions for directories.

Reply:
We should consider adding for each ob type a template relative path to the input files, and replace dates, rather than just globbing

Reply:
```
template_path = './gdas.%Y%m%d/%H/products/obstats/gdas.t%Hz.viirs_n20_iodastats.nc`
os.path.join(experiment_dir, filledin_template_dir)
```